Rakefile.dist

```diff
@@ -43,18 +44,18 @@ task :build => [
 
 task :install => :build do |t|
   mkdir_p "#{MANDIR}/man8"
-  cp "#{BUILDDIR}/foreman-rake.8.gz", "#{MANDIR}/man8/"
-  cp "#{BUILDDIR}/foreman-debug.8.gz", "#{MANDIR}/man8/"
-  cp "#{BUILDDIR}/foreman-tail.8.gz", "#{MANDIR}/man8/"
+  cp_p "#{BUILDDIR}/foreman-rake.8.gz", "#{MANDIR}/man8/"
+  cp_p "#{BUILDDIR}/foreman-debug.8.gz", "#{MANDIR}/man8/"
+  cp_p "#{BUILDDIR}/foreman-tail.8.gz", "#{MANDIR}/man8/"
 
   # foreman-tail
   TAILDIR = ENV['TAILDIR'] || "script/foreman-tail.d"
   mkdir_p "#{DATAROOTDIR}/foreman/#{TAILDIR}"
-  cp Dir.glob("#{TAILDIR}/common/*"), "#{DATAROOTDIR}/foreman/#{TAILDIR}/", :verbose => true
+  cp_p Dir.glob("#{TAILDIR}/common/*"), "#{DATAROOTDIR}/foreman/#{TAILDIR}/"
   if is_redhat
-    cp Dir.glob("#{TAILDIR}/redhat/*"), "#{DATAROOTDIR}/foreman/#{TAILDIR}/", :verbose => true
+    cp_p Dir.glob("#{TAILDIR}/redhat/*"), "#{DATAROOTDIR}/foreman/#{TAILDIR}/"
   elsif is_debian
-    cp Dir.glob("#{TAILDIR}/debian/*"), "#{DATAROOTDIR}/foreman/#{TAILDIR}/", :verbose => true
+    cp_p Dir.glob("#{TAILDIR}/debian/*"), "#{DATAROOTDIR}/foreman/#{TAILDIR}/"
```

Comment:
This should not belong here, I thought it's incorrect to not to preserve timestamps. I mean, it is, but I assume you want this out of this patch.